composer.json

```diff
@@ -37,6 +37,18 @@
   "require-dev": {
     "newfold-labs/wp-php-standards": "^1.2.3"
   },
+  "scripts": {
+    "fix": [
```

Comment:
Consider calling it `cs-fix`, per [php-pds/composer-script-names](https://github.com/php-pds/composer-script-names), similarly below.

Reply:
I don't feel very strongly on this. I like the idea of conventions but I don't particularly like the name.